VideoLocker/res/values/styles.xml

```diff
@@ -65,6 +66,12 @@
         <item name="android:textColor">@color/white</item>
     </style>
 
+    <!-- ActionBar tabs styles -->
+    <style name="MyActionBarTabs" parent="@android:style/Widget.Holo.Light.ActionBar.TabView">
```

Comment:
The "ActionBar" part should probably be removed from the name, since this will be the generic tab style, and we aren't using the action bar tabs at the moment anyway.